R/loadLm.R

```diff
@@ -88,7 +88,7 @@ loadLm <- function(formula, pred.format=c("flux","conc"),
                    data, metadata, fitting_function=NULL, 
                    y.trans.function=NULL, retrans.function=exp,
                    store=c("data","fitting.function"), ylog) {
-  
+  s.hat=NULL
```

Comment:
It's the same reason I added aes=TimeInterval=geom_histogram=xlab=NULL in the previous issue. I guess roxygen thinks it should be a global variable, seems to be some parsing issue and this circumvents that.